fendermint/vm/message/src/conv/from_eth.rs

```diff
@@ -37,7 +37,12 @@ pub fn to_fvm_message(tx: &Eip1559TransactionRequest) -> anyhow::Result<Message>
     // The `from` of the transaction is inferred from the signature.
     // As long as the client and the server use the same hashing scheme,
     // this should be usable as a delegated address.
-    let from = to_fvm_address(tx.from.unwrap_or_default());
+    let from = if let Some(from) = tx.from {
+        to_fvm_address(from)
+    } else {
+        // This is similar to https://github.com/filecoin-project/lotus/blob/master/node/impl/full/eth_utils.go#L124
```

Comment:
Good one, that way we use the same as Lotus and the behavior is somewhat predictable.